docs/concepts/python-versions.md

```diff
@@ -326,6 +328,33 @@ and the pre-release version will be used.
 If a pre-release Python version is available and matches the request, uv will not download a stable
 Python version instead.
 
+## Free-threaded Python
+
+uv supports discovering and installing
+[free-threaded](https://docs.python.org/3.14/glossary.html#term-free-threading) Python variants in
+CPython 3.13+.
+
+Free-threaded Python versions will not be selected by default. Free-threaded Python versions will
+only be selected when explicitly requested, e.g., with `3.13t` or `3.13+freethreaded`.
+
+## Debug Python variants
+
+uv supports discovering and installing
+[debug builds](https://docs.python.org/3.14/using/configure.html#debug-build) of Python, i.e., with
+debug assertions enabled. These builds also do not have debug symbols stripped, which can be useful
+when debugging a Python process with a C-level debugger.
```

Comment:
This is less accurate than the phrasing I suggested :) As I understand it, uv will both _discover_ existing `--with-pydebug` builds on the system (e.g. `dnf install python3.14-debug`, which gets you a `/usr/bin/python3.14d`), and _install_ our own managed versions, and in both cases it will only use them if it cannot find a non-debug version. But the lack of stripping of debug symbols only applies to our own debug builds and may not apply to existing discoverable versions. (And it also only applies for now, I would like to do something else about debug symbols in the future.)

Reply:
These are all great reasons not to say this at all imo, it's too complicated :)